<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


# H1: Data Science Tools and Ecosystem 

In this notebook, Data Science Tools and Ecosystem are summarized

Some of the popular languages that Data Scientists use are:

1. SQL
2. R
3. Java
4. JavaScript

Some of the commonly used libraries used by Data Scientists include:

1. Numpy
2. Panda
3. NLTK
4. LightGBM

| Data Science Tools |
| ------------------ |
| Python |
| Rstudio |
| Jypyter Notebook |

### H3: Below are a few examples of evaluating arithmetic expressions in Python.





In [24]:
# Addition:

result = 5 + 4
print(result)  # Output: 9


9


In [25]:
# Subtraction:

result = 10 - 5
print(result)  # Output: 5

5


In [26]:

# Multiplication:

result = 2 * 7
print(result)  # Output: 14

14


#  This a simple arithmetic expression to mutiply then add integers

In [27]:
(3*4)+5

17

This will convert 200 minutes to hours by diving by 60

In [28]:
200/60

3.3333333333333335

200 minutes = 3.3333333333333335 hours

**Objectives** :

List popular languages for Data Science 

- Python
- SQL
- R
- Java 


## H2 style heading: 

Author: LOK HIM LUI